File: IssueTracker.Common/Models/Comment.cs
using System;
using IssueTracker.Common.Models.Base;

namespace IssueTracker.Common.Models
{
	public class Comment : IdModel
	{
		public string Text { get; set; }
		public DateTime Date { get; set; }
		public User User { get; set; }
		public Issue Issue { get; set; }
	}
}

File: data/mapping/definitions/comment.js
var moment = require("moment");
var config = require("../../../config");

require("../mapper").define("comment", "issue-history-view-model", {
	id: "id",
	date: function(x) { return moment(x.created_at).format("YYYY-MM-DDTHH:mm:ssZ"); },
	text: "text",
	user: function(x) { return x.user; },
	userId: "userId",
	issueId: "issueId"
});

require("../mapper").define("issue-history-view-model", "comment", {
	"created_at": function(x) { return moment(x.date, config("dateTimeFormat")); },
	text: "text"
});